src/coverlet.core/Abstracts/IProcessExitHandler.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System;
+
+namespace Coverlet.Core.Abstracts
+{
+    public interface IProcessExitHandler
```

Comment:
We need this service to plug a fake in child process to avoid race condition on restore module.
In normal condition(no exception) we restore modules during coverage calculation, after  @ViktorHofer update we restore also other instrumented module on exit, on msbuild/console/collector we use a shared dictionary but on test we instrument and run on child process so parent/child race on restore/delete files.
On child test we'll do nop.